app/src/main/java/com/sun/music_64/screen/GenreScreen/StringUtil.java

```diff
@@ -0,0 +1,8 @@
+package com.sun.music_64.screen.GenreScreen;
+
+public class StringUtil {
```

Comment:
```suggestion
public class StringUtils {
```và em nên để trong package utils. em xem lại project struct